packages/core/src/modules/vc/models/credential/W3cJsonCredential.ts

```diff
@@ -10,4 +10,59 @@ export interface W3cJsonCredential {
   expirationDate?: string
   credentialSubject: SingleOrArray<JsonObject>
   [key: string]: unknown
+  credentialStatus?: SingleOrArray<CredentialStatus>
+}
+
+type CredentialStatusType = 'BitstringStatusListEntry'
```

Comment:
Can we move all the status list logic related to bit string status list to a separate file?